File: Front-end/src/app/locations/park.ts
import { BaseEntity } from '../shared/baseEntity';
import { Company } from '../locations/company';
import { Country } from '../lookups/country';
import { City } from '../lookups/city';
import { Lookup } from '../lookups/lookup';
import { Rating } from '../shared/enums/rating';

export interface Park extends BaseEntity {
		/** public int ParkId { get; set; } */
		name: string;
		address: string;
		place: {
			country: Country;
			state: {
				stateId: number;
				name: string;
			};
			city: City;
			other: string;
		};
		location: Location;
		rating: Rating;
		establishedYear?: number;
		area?: number;
		companies: Company[];
		industries: Lookup[];
}

File: Service/Z2data.Core/Domain/Locations/Park.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Z2data.Core.Domain.Events;
using Z2data.Core.Domain.Lookups;
using Z2data.Core.Enums;

namespace Z2data.Core.Domain.Locations
{
    public class Park : BaseEntity
    {
        //public int ParkId { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public Place Place { get; set; }
        public Location Location { get; set; }
        public Rating Rating { get; set; }
        public int? EstablishedYear { get; set; }
        public double? Area { get; set; }
        public IList<Company> Companies { get; set; }
        public IList<Lookup> Industries { get; set; }
    }
}
